sqlmesh/core/config/root.py

```diff
@@ -40,9 +39,13 @@ class Config(BaseConfig):
         auto_categorize_changes: Indicates whether SQLMesh should attempt to automatically categorize model changes (breaking / non-breaking)
             during plan creation.
         users: A list of users that can be used for approvals/notifications.
+        username: Name of a single user who should receive approvals/notification, instead of all users in the `users` list.
         pinned_environments: A list of development environment names that should not be deleted by the janitor task.
+        loader: Loader class used create a SQLMesh context from project files.
+        env_vars: A dictionary of environmental variable names and values.
         model_defaults: Default values for model definitions.
         include_unmodified: Indicates whether to include unmodified models in the target development environment.
+        physical_schema_override: A dictionary of model schema names and the physical schemas SQLMesh should use for the models' physical tables.
```

Comment:
"A mapping from model schema names to names of schemas in which physical tables for corresponding models will be placed"